.github/workflows/ci-cd.yml

```diff
@@ -1318,21 +1292,27 @@ jobs:
     strategy:
       matrix:
         target-container:
-        - tag: fedora:35
-        - tag: fedora:36
         - tag: fedora:37
+        - tag: fedora:38
+        - tag: fedora:39
         - tag: centos/centos:stream8
           registry: quay.io
-        - tag: ubi8/ubi:8.3
-          registry: registry.access.redhat.com
-        - tag: ubi8/ubi:8.4
-          registry: registry.access.redhat.com
         - tag: ubi8/ubi:8.5
           registry: registry.access.redhat.com
         - tag: ubi8/ubi:8.6
           registry: registry.access.redhat.com
+        - tag: ubi8/ubi:8.7
+          registry: registry.access.redhat.com
+        - tag: ubi8/ubi:8.8
```

Comment:
Looks like UBI 8 only ships with Python 3.6:
https://github.com/ansible/pylibssh/actions/runs/6788921335/job/18455029172?pr=533#step:20:622

Reply:
Yeah, that does seem to be a problem. I can try to walk back the Python version raise or drop ubi8, not sure if there are other options available.

Reply:
Okay, we can punt the 3.6 support issue for now. I didn't really want that in this PR anyway.

Some discussion of what we are testing here and how many versions to keep might not be amiss at this point.